packages/react-native-reanimated/Common/cpp/Fabric/ReanimatedCommitHook.cpp

```diff
@@ -38,23 +40,21 @@ RootShadowNode::Unshared ReanimatedCommitHook::shadowTreeWillCommit(
   // ShadowTree not commited by Reanimated, apply updates from PropsRegistry
 
   auto rootNode = newRootShadowNode->ShadowNode::clone(ShadowNodeFragment{});
+  std::unordered_map<
+      const ShadowNodeFamily *,
```

Comment:
It doesn't seem safe to store pointers to `ShadowNodeFamily`, is there any alternative? If that's not possible, can we at least hold references instead of pointers (using `std::reference_wrapper`)?

Reply:
I only use `ShadowNodeFamily` whilst still holding onto corresponding `ShadowNodes`. Since `ShadowNode` owns its `family` through a `shared_ptr` it should be safe.